AUTHORS

```diff
@@ -32,14 +36,15 @@ Ivan Mitrichev, Mendeleev University of Chemical Technology of Russia
 Harry Moffat, Sandia National Laboratory
 Christopher Neal
 Kyle Niemeyer, Oregon State University
-Paul Northrop
-Andreas Rücker
+Paul Northrop (@pwcnorthrop)
+Andreas Rücker (@cannondale1492)
```

Comment:
If you think we should be linking to GH usernames here, why only for some users?

Reply:
I added them for non-obvious usernames. Based on your feedback, there are two options: use GH handle if full name is not provided on GH profile, or add GH handle across the board?

Reply:
Yes, I would see those as the two options -- it's not really obvious that the GH handles for the other contributors are more obviously based on their names. I would lean toward the second choice, since I don't think these will get  automagically parsed as links to GH profiles anywhere (or at least, https://github.com/Cantera/cantera/blob/master/AUTHORS doesn't do it), and if you want to see contributions associated with GH accounts, https://github.com/Cantera/cantera/graphs/contributors is going to be a much better bet.

Reply:
I added GH handles for everyone listed on the Github contributor list.